regression-test/suites/load/broker_load/test_array_load.groovy

```diff
@@ -116,13 +137,30 @@ suite("test_array_load", "load") {
         assertTrue(result1[0][0] == 0, "Query OK, 0 rows affected")
     }
     
+    def check_load_result = {checklabel, testTablex ->
+        max_try_micro_secs = 10000
```

Comment:
```suggestion
        max_try_milli_secs = 10000
```

Reply:
this is a good suggestion. I will update it.